e2e/cypress/integration/notifications/browser_tab_notification_spec.js

```diff
@@ -0,0 +1,144 @@
+
+        // * Check for no unreads or mentions
+        cy.get('.unread-title').should('not.exist');
+
+        // # Save path of current favicon path
+        cy.get('link[rel=icon]').should('have.attr', 'href').then(($path) => {
+            originalFavicon = $path;
+        });
+
+        // # Have another user view team B
+        cy.apiLogin(user2);
+        cy.visit(testTeam2TownSquareUrl);
+        cy.wait(TIMEOUTS.FIVE_SEC);
+
+        // # Post a message with mentioning User 1 from Town Square
+        cy.postMessage(`@${user1.username}`);
+
+        // # Return to User 1
+        cy.visit(testTeam1TownSquareUrl);
+        cy.apiLogin(user1);
+
+        // * Browser tab should displays (1) * channel - [team name] Mattermost
+        cy.title().should('include', '(1)');
+
+        // * Team sidebar: Small dot left of team B icon in team sidebar
+        cy.get(`#${team2.name}TeamButton`).parent('.unread').should('be.visible').
+            within(() => {
+                // * Team sidebar: a mention badge in top right corner of the badge with number "1"
+                cy.get('.badge').contains('1').
+
+                // # Make mention indicated
+                    click();
+            });
+
+        // * Favicon should stay blue (not turn red) when there are mentions indicated
+        cy.get('link[rel=icon]').should('have.attr', 'href').then(($path) => {
+            expect($path).to.eq(originalFavicon);
+        });
+    });
+
+    it('MM-T560_2 Team sidebar icon - Badge with mention count increments when added to channel', () => {
+        // # User 1 view and remain on team A
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+
+        // # Have another user view team B
+        cy.apiLogin(user2);
+        cy.visit(testTeam2TownSquareUrl);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Create a new channel
+        cy.get('#createPublicChannel').should('be.visible').click();
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('#newChannelName').should('be.visible').type('new-channel');
+        cy.get('#submitNewChannel').click();
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Invites User 1
+        cy.get('#member_popover').should('be.visible').click();
+        cy.contains('Manage Members').click();
+        cy.contains('Add Members').click();
+        cy.contains(`${user1.username}`).click();
+        cy.get('#saveItems').click();
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // * User 1 should be added.
+        cy.getLastPost().
+            should('contain', `@${user1.username}`).
+            should('contain', 'added to the channel');
+
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Change User 2's view to Team 1
+        cy.visit(testTeam1TownSquareUrl);
+
+        // # Switch to User 1
+        cy.apiLogin(user1);
+
+        // Because user changed by api, refresh is needed
+        cy.visit(testTeam1TownSquareUrl);
+        cy.wait(TIMEOUTS.HALF_SEC);
```

Comment:
I suggest to simplify this like,
```suggestion
        // # Switch to User 1 and visit the town-square
        cy.apiLogout();
        cy.apiLogin(user1);
        cy.visit(testTeam1TownSquareUrl);
```